src/index.module.scss

```diff
@@ -898,7 +898,7 @@ span.software {
 .nc-detail-wrapper {
   padding: 1rem;
   border: 1px solid;
-  max-width: 100%;
+  width: 100%;
```

Comment:
The screen was really bad with data that had small width. I could've added a `min-width: 80%` but I preferred keeping it fixed as 100%